docs/developer/cluster-lifecycle-api-provider-integration.md

```diff
@@ -0,0 +1,60 @@
+# Cluster Lifecycle API provider integration guide
+
+> `THIS IS A WORK IN PROGRESS`
+
+## Context
+> Note: if you haven't yet, reading the [original design doc](https://github.com/aws/eks-anywhere/blob/main/designs/full-cluster-lifecycle-api.md) is a must to execute this integration.
```

Comment:
is the expectation to continue to support CLI-based operations as well, or stop supporting that in favor of controller-based operations?

Reply:
Great question

TBD if we continue to support CLI commands to manage workload clusters. My guess is that we will. However, we should change the internal implementation of CLI to make use of the controller, by just applying the changes with kubectl.